db/client.js

```diff
@@ -0,0 +1,8 @@
+// Connect to DB
+const { Client } = require("pg");
+
+const CONGO = "shop-db";
```

Comment:
include everything as a environmental variable, if possible, including the database name